### PR TITLE
Fix test initialization logic which is broken after App.java was restructured

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/config/Config.java [hunk 1]
```diff
@@ -81,18 +82,31 @@ public FutureJdbi getJdbi() {
...
+      return initializeJdbi(databaseConfig);
+    }
+    return this.jdbi;
+  }
 
-      final Jdbi jdbi = Jdbi.create(hikariDataSource);
-      final Executor dbExecutor = FutureGrpc.initializeExecutor(databaseConfig.threadCount);
-      this.jdbi = new FutureJdbi(jdbi, dbExecutor);
+  public FutureJdbi getTestJdbi() {
+    if (this.jdbi == null) {
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/config/Config.java:91]: Is it safe to use the exact same object?

File: backend/src/main/java/ai/verta/modeldb/config/Config.java [hunk 2]
```diff
@@ -81,18 +82,31 @@ public FutureJdbi getJdbi() {
...
+    final var dbUrl = CommonHibernateUtil.buildConnectionString(databaseConfig.RdbConfiguration);
+    hikariDataSource.setJdbcUrl(dbUrl);
+    hikariDataSource.setUsername(databaseConfig.RdbConfiguration.RdbUsername);
+    hikariDataSource.setPassword(databaseConfig.RdbConfiguration.RdbPassword);
+
+    hikariDataSource.setMaximumPoolSize(10); // TODO: configure with a valid pool size
+
+    final Jdbi jdbi = Jdbi.create(hikariDataSource);
+    final Executor dbExecutor = FutureGrpc.initializeExecutor(databaseConfig.threadCount);
+    return new FutureJdbi(jdbi, dbExecutor);
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/config/Config.java:110]: We're no longer saving `this.jdbi` anywhere. We create a new instance every single time the `get` methods are called.

**Review by conradoverta — CHANGES_REQUESTED** [review]

File: backend/src/main/java/ai/verta/modeldb/config/Config.java (hunk 2 above)
Comment (ad-47) [reply backend/src/main/java/ai/verta/modeldb/config/Config.java:110]: We are checking `if (this.jdbi == null) {` before calling `initializeJdbi` method so if it was already initialized then we are return existing one.

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/config/Config.java:110]: Yes. fixed this. it is a bug.

File: backend/src/main/java/ai/verta/modeldb/config/Config.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/config/Config.java:91]: Yes. we are used same `jdbi` but used test config so yes it is safe.

**Review by conradoverta — APPROVED** [review]

**Pull request merged**